### PR TITLE
Skip third_party/benchmark in the license source crawl

### PR DESCRIPTION
**Pull request by jason-simmons**
Also regenerate sky_engine/LICENSE to include libtxt

### CODE REVIEW

**Comment by jason-simmons** [comment]
@Hixie @B3rn475

File: sky/packages/sky_engine/LICENSE [hunk 1]
```diff
@@ -9194,6 +9194,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 etc1
 observatory_pub_packages
 skia
+txt
```
Comment (Hixie) [inline sky/packages/sky_engine/LICENSE]: presumably the golden files need updating too, if this change is real?

**Comment by Hixie** [comment]
LGTM for the dart change, the license change is either incomplete or extraneous.

**Comment by Hixie** [comment]
I'm confused. The change to sky/packages/sky_engine/LICENSE in this patch doesn't match the change to the golden files.

**Pull request merged**